package-lock.json

```diff
@@ -1,398 +1,4671 @@
 {
-  "name": "giantswarm-v4",
+  "name": "giantswarm-",
```

Comment:
Is the trailing - intentional here?

Reply:
That's because for some reason it was like that in `package.json`. I updated it, though